doc/api/crypto.md

```diff
@@ -2659,6 +2659,22 @@ added: v10.0.0
 Enables the FIPS compliant crypto provider in a FIPS-enabled Node.js build.
 Throws an error if FIPS mode is not available.
 
+### crypto.signOneShot(data, key)
```

Comment:
Needs to be adapted to the new parameters.

Reply:
Fixed.